$OpenBSD: patch-cmake_modules_FindPhonon_cmake,v 1.2 2013/12/08 19:13:32 zhuk Exp $
Make sure Phonon is found, and make sure it is found in private directory.
--- cmake/modules/FindPhonon.cmake.orig	Fri Jun 28 21:03:40 2013
+++ cmake/modules/FindPhonon.cmake	Sun Oct 13 19:03:12 2013
@@ -22,7 +22,7 @@ macro(_phonon_find_version)
 endmacro(_phonon_find_version)
 
 # the dirs listed with HINTS are searched before the default sets of dirs
-find_library(PHONON_LIBRARY NAMES phonon HINTS ${KDE4_LIB_INSTALL_DIR} ${QT_LIBRARY_DIR})
+find_library(PHONON_LIBRARY NAMES phonon_s phonon HINTS ${KDE4_LIB_DIR} ${KDE4_LIB_INSTALL_DIR} ${QT_LIBRARY_DIR})
 find_path(PHONON_INCLUDE_DIR NAMES phonon/phonon_export.h HINTS ${KDE4_INCLUDE_INSTALL_DIR} ${QT_INCLUDE_DIR} ${INCLUDE_INSTALL_DIR} ${QT_LIBRARY_DIR})
 
 if(PHONON_INCLUDE_DIR AND PHONON_LIBRARY)
